app/jobs/geocode_location_job.rb

```diff
@@ -47,6 +51,6 @@ def enqueue_next_job
   end
 
   def next_location
-    Location.where('latitude IS NULL OR longitude IS NULL').first
+    Location.where(geocode_response: nil).first
```

Comment:
Hmm. How do you see the automatic retry behavior starting? I was planning on failed geocode responses not being nil, because they would have this error: https://github.com/artsy/bearden/pull/276/files#diff-462e4ddf943cd6a3904178ecc6528df4R45.